src/plugins/ui/evaluation_chart.ts

```diff
@@ -30,7 +30,9 @@ export class EvaluationChartPlugin extends UIPlugin {
     if (
       invalidateEvaluationCommands.has(cmd.type) ||
       cmd.type === "EVALUATE_CELLS" ||
-      (cmd.type === "UPDATE_CELL" && "content" in cmd)
+      (cmd.type === "UPDATE_CELL" && "content" in cmd) ||
+      cmd.type === "UNHIDE_COLUMNS_ROWS" ||
+      cmd.type === "HIDE_COLUMNS_ROWS"
```

Comment:
Note for forward ports: do'nt forget other ways to hide headers (data filters, header groups) + add test